test/e2e/serial/workload_placement_test.go

```diff
@@ -561,6 +561,84 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 		})
 	})
 
+	Context("with a no suitable node", func() {
+		var requiredRes corev1.ResourceList
+		BeforeEach(func() {
+			requiredNUMAZones := 2
+			By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", requiredNUMAZones))
+			nrtCandidates := e2enrt.FilterZoneCountEqual(nrts, requiredNUMAZones)
+
+			const neededNodes = 1
```

Comment:
let's either use `const` also above on line 567 or drop it from here.
I for myself I still have a funky feeling about using `const` this way :\